custom/plugins/VirtuaShipping/Components/ShippingService.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
```

Comment:
Please use "&&" instead of "and". It's more common, easier to read by php developer and it's used by default in shopware